content/kots-cli/reset-password.md

```diff
@@ -6,3 +6,21 @@ weight: 90160
 ---
 
 If you’ve deployed an application with the admin console, the `kots reset-password` command will change the bcrypted password hash in the cluster, allowing you to log in again.
+
+### Usage
+```bash
+kubectl kots reset-password [namespace] [flags]
+```
+* _Replace `[namespace]` with the namespace where your KOTS application resides_ 
```

Comment:
Specifically, it's not where the application resides, but where the Admin Console is deployed

Reply:
Will update wording.